### DISCUSSION TITLE
Can anyone estimate their internet traffic to BirdWeather?

### DISCUSSION THREAD

**Discussion by w0mbatian** [mcguirepr89/BirdNET-Pi#557] [Q&A] [answered]
I have just got BirdNET-Pi running and have just received my BirdWeather ID.
However I live in a remote area of Australia which is great for birds but that means my internet comes via satellite and is limited and fairly expensive. So before I start with BirdWeather I would like to get some idea of the traffic that might generate on my internet connection. Upload is counted as well.
I imagine mine will be a fairly busy BirdNET-Pi installation. I have had 42 detections in the last hour and 22 species today and it is only 10-30 AM.
I would just try for myself but this has been a very big month for usage and we are running really short.
Hundreds of megabytes per month would be fine but gigabytes per month would be too much.
Just a vague ball park estimate from your experience will be fine.
Ian

**Comment by mcguirepr89** [comment, marked as answer]
Hi, @w0mbatian -- I would say for your situation you will want to pass on BirdWeather -- it uploads the entire recording (not just the identified chunk) to BirdWeather's API and will result in a _very_ substantial amount of upload bandwidth.

**Comment by w0mbatian** [comment]
Oh that is disappointing. I love taking part in community projects.
Perhaps some consideration could be given in the future to ways of limiting or controlling what is uploaded so that people in my situation could still take part?
